docs/book/deployment.md

```diff
@@ -451,13 +469,13 @@ Once again, you can view the result skill invocation names in the deployed model
 
 Please see the [Monetization chapter](/book/monetization.html).
 
-## Extra Note: The `production` Deployment Target Name
+## Extra Note: Including `production` in the Deployment Target Name
 
-There is a special deployment target name called `production`.
-For deployment targets that are *not* this, a `\ (development)` will be
-automatically appended to the name of your skill, with `development`
-specified as your named deployment target. We recommend creating and
-using this deployment target for your live skill.
+There is a special condition on deployment target names. If the name does *not*
+have the word `production` in it, a ` (development)` will be automatically
+appended to the name of your skill, with `development` specified as your named
```

Comment:
Would reword to more clearly indicate that the deployment target name (in this case 'development') is what's appended in parentheses.